pandas/tests/reshape/test_pivot.py

```diff
@@ -142,6 +142,25 @@ def test_pivot_table_dropna_categoricals(self, dropna):
 
         tm.assert_frame_equal(result, expected)
 
+    def test_pivot_with_new_patch(self, dropna):
+        # gh-21133
```

Comment:
can you just add this example into an existing test (the one below looks like a candidate).